src/DataStructures/DataBox/TagTraits.hpp

```diff
@@ -21,18 +21,15 @@ namespace db {
 /*!
  * \ingroup DataBoxGroup
  * \brief Check if `Tag` derives off of db::ComputeTag.
+ *
  * \see is_compute_tag_v ComputeTag
  */
-template <typename Tag, typename = std::nullptr_t>
-struct is_compute_tag : std::false_type {};
-/// \cond
 template <typename Tag>
-struct is_compute_tag<Tag, Requires<std::is_base_of_v<db::ComputeTag, Tag>>>
-    : std::true_type {};
-/// \endcond
+struct is_compute_tag
+    : std::bool_constant<std::is_base_of_v<db::ComputeTag, Tag>> {};
```

Comment:
isn't this just `: std::is_base_of<db::ComputeTag, Tag>`? No need to convert to a bool constant I think